app/controllers/dealer_histories_controller.rb

```diff
@@ -0,0 +1,59 @@
+class DealerHistoriesController < ApplicationController
+  before_action :set_dealer_history, only: :show
+
+  def index
+    @dealer_histories = DealerHistory.where(dealer_id: dealer.id)
+    render json: { dealer_histories: @dealer_histories }, status: :ok
```

Comment:
better if we make it same like in the `PlayerHistoriesController`
```suggestion
    render json: { histories: @dealer_histories }, status: :ok
```